src/BaGetter.Database.SqlServer/BaGetter.Database.SqlServer.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" />
+    <PackageReference Include="Microsoft.Data.SqlClient" />
```

Comment:
Please add a comment like `Can be removed once .SqlServer updates its own dependency beyond 5.1.3" so someone in the future maybe thinks to remove it when possible.